app/experimenter/kinto/tasks.py

```diff
@@ -296,7 +296,9 @@ def nimbus_check_experiments_are_paused():
                 experiment.is_paused = True
                 experiment.save()
 
-                generate_nimbus_changelog(experiment, get_kinto_user())
+                generate_nimbus_changelog(
+                    experiment, get_kinto_user(), "is_paused is set to True"
```

Comment:
Eventually these will show up in a history page, so we might want to make them more end user consumable like 'Enrollment was paused' or something?

Reply:
Yeah, that makes more sense. I had something like that earlier, but figured I should try to follow the other log messages in that neighborhood.

Reply:
Yeah if others in that area are too technical we should make them more human readable too, but the ones you put for created/deleted/etc all looked fine 👍

Reply:
And we can change these messages later so it's not the end of the world